README.md

```diff
@@ -2,15 +2,52 @@
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ome/omero-analysis-desktop/master?filepath=napari.ipynb)
 
 
-Run [OMERO clients](https://www.openmicroscopy.org/omero/downloads/) and [Napari](http://napari.org/) in a Linux desktop using Jupyter.
+Run [OMERO clients](https://www.openmicroscopy.org/omero/downloads/), [Napari](http://napari.org/) and [omero-napari](https://gitlab.com/openmicroscopy/incubator/omero-napari) in a Linux desktop using Jupyter.
 
 This is based on https://github.com/ryanlovett/nbnovnc
 
 Either [run on mybinder.org](https://mybinder.org/v2/gh/jburel/omero-analysis-desktop/master) or build locally with [repo2docker](https://repo2docker.readthedocs.io/):
+
+To build locally:
+
+ * Install [Docker](https://www.docker.com/) if required
+ * Create a virtual environment and install repo2docker from PyPI.
+ * Clone this repository
+ * Run  ``repo2docker``. 
+ * Depending on the permissions, you might have to run the command as an admin
+
 ```
+pip install jupyter-repo2docker
+git clone https://github.com/ome/omero-analysis-desktop.git
+cd omero-analysis-desktop
 repo2docker .
 ```
 
-Open the displayed URL, then go to `/desktop` e.g. http://localhost:8888/desktop and you should see a Linux desktop with icons for OMERO.insight and FIJI.
+Open the displayed URL, then go to `/desktop` in the menu option on the right-hand side.
+You should see a Linux desktop with icons for OMERO.insight and FIJI.
+
+Use omero-napari
+~~~~~~~~~~~~~~~~
+
```

Comment:
Something's gone wrong with the formatting in this bit:

![image](https://user-images.githubusercontent.com/1644105/80583958-e4ba2f80-8a08-11ea-8443-c4a1128f810d.png)